app/concepts/api/v1/serializer/error.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Api::V1::Serializer
+  class Error
+    def initialize(object)
+      @object = object
+    end
+
+    def to_json(*_args)
+      if @object.is_a?(Dry::Validation::Result)
```

Comment:
For all cases we'll pass `Dry::Validation::Result` instances only. Please simplify.

Reply:
Ok